amortization.py

```diff
@@ -231,8 +254,38 @@ def __str__(self):
                    (self.rate, self.nper, self.pv, self.typ))
 
 
+    def _calc_pmt_diff_(self, pmt):
+        """ Difference between last payment and penultimate payment """
+        l = Loan(self.rate, self.nper, self.pv, self.date, payment=pmt)
+        return l.period(l.nper).payment - l.period(l.nper-1).payment
+
     def pmt(self):
-        return self.payment or pmt(self.rate, self.nper, self.pv, self.typ)
+        if self.payment:
+            return self.payment
+
+        if PMT_METHOD == ORDINARY_ANNUITY_PMT_METHOD:
```

Comment:
I'll put a comment in here explaining we're trying to smooth out the semi-monthly payments